src/v2/ion/ui-schema/ion-string-handler.js

```diff
@@ -91,7 +110,8 @@ const createUiSchemaForString = (ionFormField, remediationForm, transformedResp,
   }
 
   // set optional label for text boxes
-  if(ionFormField.required === false && uiSchema.type === 'text') {
+  if(ionFormField.required === false && (uiSchema.type === 'text' || uiSchema.type === 'radio' ||
```

Comment:
lets create an array `optionalType` for this `uiSchema.type === 'text' || uiSchema.type === 'radio' ||
      uiSchema.type === 'select'` 

and check .. `optionalType.includes(uiSchema.type)`

Also modify the comment above

Reply:
I have modified the comment above and created the array `optionalType`